persistence/javadsl/src/main/java/com/lightbend/lagom/javadsl/persistence/PersistentEntityRef.java

```diff
@@ -24,12 +25,12 @@
 public final class PersistentEntityRef<Command> implements NoSerializationVerificationNeeded {
   private final String entityId;
   private final ActorRef region;
-  private final Timeout timeout;
+  private final Duration timeout;
 
   public PersistentEntityRef(String entityId, ActorRef region, FiniteDuration askTimeout) {
     this.entityId = entityId;
     this.region = region;
-    this.timeout = Timeout.apply(askTimeout);
+    this.timeout = Duration.ofNanos(askTimeout.toNanos());
```

Comment:
`PatternsCS` is deprecated in Akka 2.5.19

Now users are requested to use `akka.pattern.Patterns` and there is a new `ask` method that receives a `java.time.Duration` and returns `CompletionStage` to be used by Java APIs. 

`akka.pattern.Patterns` has the following now: 

```scala
def ask(actor: ActorRef, message: Any, timeout: Timeout): Future[AnyRef]
def ask(actor: ActorRef, message: Any, timeout: java.time.Duration): CompletionState[AnyRef]
```

Since we are not using `PatternsCS` anyrmore, we need to pass a `Duration` in order to get back a `CompletionStage`.

In this PR I'm trying to keep it minimal, but we probably want to follow the same strategy and stop to use `FiniteDuration` in the Java API.

Reply:
See #1673.